hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/hedera/subprocess/SubProcessNetwork.java

```diff
@@ -850,11 +855,13 @@ private int findAvailablePort() {
         throw new RuntimeException("Could not find available port after 100 attempts");
     }
 
-    public static long shardFrom(final List<HederaNode> nodes) {
-        return requireNonNull(nodes.getFirst()).getAccountId().shardNum();
+    @Override
+    public long shard() {
+        return shard;
```

Comment:
Thank you for finding and fixing this! I had totally missed that `AbstractNetwork` already has a `shard()` and `realm()` method